rule/struct_tag.go

```diff
@@ -358,6 +364,21 @@ func (w lintStructTagRule) checkURLTag(options []string) (string, bool) {
 	return "", true
 }
 
+func (w lintStructTagRule) checkDatastoreTag(options []string) (string, bool) {
+	for _, opt := range options {
+		switch opt {
+		case "flatten", "noindex", "omitempty":
+		default:
+			if w.isUserDefined(keyDatastore, opt) {
+				continue
+			}
+			return fmt.Sprintf("unknown option '%s' in Datastore tag", opt), false
```

Comment:
Nitpicking

```suggestion
			return fmt.Sprintf("unknown option '%s' in datastore tag", opt), false
```

Or a variation of this, with quotes maybe.

I'm unsure google datastore tag should be considered as an acronym or a brand.

JSON, BSON, XML were acronym. But look at the way "protobuf" tag is reported.

Reply:
I have also hesitated with the name :)
The [web site](https://cloud.google.com/products/datastore) says "Datastore"

Anyway, I'll write it all lowercases